mappings/net/minecraft/world/poi/PointOfInterestType.mapping

```diff
@@ -2,8 +2,11 @@ CLASS net/minecraft/class_4158 net/minecraft/world/poi/PointOfInterestType
 	FIELD comp_815 blockStates Ljava/util/Set;
 	FIELD comp_816 ticketCount I
 	FIELD comp_817 searchDistance I
+	FIELD field_39277 FALSE Ljava/util/function/Predicate;
```

Comment:
Hmm, how will this be used? To indicate missing associatable POI? If so `MISSING` would be better.